### PR TITLE
remove holding message from non-uk page and add bursary content

### PR DESCRIPTION
**Pull request by emmaachesongray**
### Trello card
https://trello.com/c/aFtKp6z8/6687

### CODE REVIEW

File: config/values/bursaries.yml [hunk 1]
```diff
@@ -16,6 +16,7 @@ bursaries:
     languagesother: £26,000
     maths: £29,000
     music: £10,000
+    nonuk: £31,000
```
Comment (gemmadallmandfe) [inline config/values/bursaries.yml:19]: would suggest putting this at the bottom of the list (as previously) as it is not really a subject!

File: app/views/content/non-uk-teachers/get-an-international-relocation-payment.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ inset_text:
         <br>
         <h2>Changes to eligibility and payment</h2>
 
-      <p>Trainees who train to teach in the 2024 to 2025 academic year are no longer eligible for the IRP. This applies to all trainees who started courses from 1 July 2024 onwards. Trainees in languages or physics may still be eligible to <a href="/funding-and-support/scholarships-and-bursaries">get a bursary or apply for a scholarship</a>.</p>
+      <p>Trainees who train to teach in the 2024 to 2025 academic year are no longer eligible for the IRP. This applies to all trainees who started courses from 1 July 2024 onwards. Trainees in languages or physics may still be eligible to <a href="/funding-and-support/scholarships-and-bursaries">get a bursary or apply for a scholarship</a></a> worth up to $scholarships_nonuk_max$.</p>
```
Comment (gemmadallmandfe) [inline app/views/content/non-uk-teachers/get-an-international-relocation-payment.md:27]: @Sarah-DfE Flagging to you that Emma is including this para in her PR so you will not need to do so in https://github.com/DFE-Digital/get-into-teaching-app/pull/4236

You will need to merge in Emma's changes once hers has gone out to pick this up?

File: config/values/bursaries.yml (hunk 1 above)
Comment (gemmadallmandfe) [inline config/values/bursaries.yml:19]:
```suggestion
    nonuk: £29,000
```
Only scholarships go up to £31k - the highest bursary is £29k?

File: app/views/content/non-uk-teachers/fees-and-funding-for-non-uk-trainees.md [hunk 3]
```diff
@@ -114,12 +110,10 @@ You will also need a [visa or immigration status](/non-uk-teachers/visas-for-non
 
 ## Bursaries and scholarships for non-UK trainee languages and physics teachers
 
-$funding-holding-message$
-
 There are 2 types of financial support you can get if you’re interested in training to teach languages or physics:
 
-* bursaries for languages and physics
-* scholarships for French, German, Spanish and physics
+* bursaries worth up to $bursaries_postgraduate_nonuk$ for languages and physics
```
Comment (gemmadallmandfe) [inline app/views/content/non-uk-teachers/fees-and-funding-for-non-uk-trainees.md:115]: Amount is displaying incorrectly due to incorrect value in `/values/bursaries` - no change needed here but just flagging for retesting

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me

**Review by Sarah-DfE — APPROVED** [review]
Looks good to me!

**Pull request merged**